tests/middleware/tests.py

```diff
@@ -750,10 +749,8 @@ def test_defaults_sameorigin(self):
         If the X_FRAME_OPTIONS setting is not set then it defaults to
         DENY.
         """
-        with override_settings(X_FRAME_OPTIONS=None):
-            del settings.X_FRAME_OPTIONS  # restored by override_settings
```

Comment:
This line was testing an artificial situation where `del` was used to remove the setting entirely. If we want to support that, I think we need to support it for *all* settings access, which would be too much.